src/OmniSharp.DotNetTest/Legacy/LegacyTestManager.cs

```diff
@@ -63,22 +71,42 @@ public override RunDotNetTestResponse RunTest(string methodName, string testFram
                 WorkingDirectory = WorkingDirectory,
                 CreateNoWindow = true,
                 UseShellExecute = false,
-                RedirectStandardOutput = false,
-                RedirectStandardError = false
+                RedirectStandardOutput = true,
+                RedirectStandardError = true
             };
 
             var testProcess = Process.Start(startInfo);
 
-            var results = new List<TestResult>();
+            var output = new StringBuilder();
+            var error = new StringBuilder();
+
+            testProcess.OutputDataReceived += (_, e) =>
```

Comment:
👍 

I hadn't jumped to the docs yet.  I just remember my experience with node that it wasn't always the case (of course we're talking two completely different tech stacks 😄)